src/Form/ContentSelectForm.php

```diff
@@ -39,12 +39,14 @@ public function buildForm(array $form, FormStateInterface $form_state, array &$i
       '#options' => $install_options,
     ];
 
-    $form['settings'] = [
-      '#type' => 'checkboxes',
-      '#title' => $this->t('Choose settings to import'),
-      '#default_value' => array_keys($settings_options),
-      '#options' => $settings_options,
-    ];
+    if (\Drupal::state()->get('openy_preset') != 'standard') {
```

Comment:
Let's remove this setting at all and install demo credentials every time we install Add this demo content.